formatter_test.go

```diff
@@ -1,145 +1,165 @@
-	expectedStateMachine := `{"StartAt": "S0", "States": {"S0": {"Type": "Map" , "Iterator": {"StartAt": "SI0", "States": {"SI0": {"Type": "Pass", "End": true}}}, "End": true}}}`
-	actualStateMachine := Format(stateMachineSource, "")
-	assert.Equal(t, expectedStateMachine, actualStateMachine)
-}
 
-func TestStrFormatWithIndexOutOfArgsRange(t *testing.T) {
-	template := "{3} - rings to the immortal elfs, {7} to dwarfs, {9} to greedy people and {1} to control everything"
-	expectedResult := "3 - rings to the immortal elfs, {7} to dwarfs, {9} to greedy people and 1 to control everything"
-	actualResult := Format(template, "0", "1", "2", "3")
-	assert.Equal(t, expectedResult, actualResult)
-}
-
-func TestStrFormatComplexKeyNotFound(t *testing.T) {
-	template := "Hello: {username}, you earn {amount} $"
-	expectedResult := "Hello: {username}, you earn 1000 $"
-	actualResult := FormatComplex(template, map[string]interface{}{"amount": 1000})
-	assert.Equal(t, expectedResult, actualResult)
-}
-
-func TestStrFormatGeneric(t *testing.T) {
-	strFormat1 := "Here we are testing integers \"int8\": {0}, \"int16\": {1}, \"int32\": {2}, \"int64\": {3} and finally \"int\": {4}"
-	var v1 int8 = 8
-	var v2 int16 = -16
-	var v3 int32 = 32
-	var v4 int64 = -64
-	var v5 int = 123
-
-	strFormatResult := Format(strFormat1, v1, v2, v3, v4, v5)
-	assert.Equal(t, "Here we are testing integers \"int8\": 8, \"int16\": -16, \"int32\": 32, \"int64\": -64 and finally \"int\": 123", strFormatResult)
-
-	strFormat2 := "Here we are testing integers \"uint8\": {0}, \"uint16\": {1}, \"uint32\": {2}, \"uint64\": {3} and finally \"uint\": {4}"
-	var v6 uint8 = 8
-	var v7 uint16 = 16
-	var v8 uint32 = 32
-	var v9 uint64 = 64
-	var v10 uint = 128
-
-	strFormatResult = Format(strFormat2, v6, v7, v8, v9, v10)
-	assert.Equal(t, "Here we are testing integers \"uint8\": 8, \"uint16\": 16, \"uint32\": 32, \"uint64\": 64 and finally \"uint\": 128", strFormatResult)
+	"github.com/stretchr/testify/assert"
 
-	strFormat3 := "Here we are testing floats \"float32\": {0}, \"float64\":{1}"
-	var v11 float32 = 1.24
-	var v12 float64 = 1.56
-	strFormatResult = Format(strFormat3, v11, v12)
-	assert.Equal(t, "Here we are testing floats \"float32\": 1.24, \"float64\":1.56", strFormatResult)
+	"github.com/wissance/stringFormatter"
+)
 
-	strFormat4 := "Here we are testing \"bool\" args: {0}, {1}"
-	var v13 bool = false
-	var v14 bool = true
-	strFormatResult = Format(strFormat4, v13, v14)
-	assert.Equal(t, "Here we are testing \"bool\" args: false, true", strFormatResult)
+const _address = "grpcs://127.0.0.1"
 
-	strFormat5 := "Here we are testing \"complex64\" {0} and \"complex128\": {1}"
-	var v15 complex64 = complex(1.0, 6.0)
-	var v16 complex128 = complex(2.3, 3.2)
-	strFormatResult = Format(strFormat5, v15, v16)
-	assert.Equal(t, "Here we are testing \"complex64\" (1+6i) and \"complex128\": (2.3+3.2i)", strFormatResult)
+type Example struct {
```

Comment:
I suggest to rename `type Example struct` to `type meteoData struct`

Reply:
@rprtr258, checked you changes, great job, but there is a minor code review issue

Reply:
renamed to `meteoData`, also, as i know, nothing from `*_test` packages is exported and doesn't even have to compile, so no worry, to leak that struct even if it was exported